packages/qcow-format/qcow-format.0.4.2/opam

```diff
@@ -21,7 +21,7 @@ depends: [
   "mirage-types-lwt" {< "3.0.0"}
   "lwt"
   "mirage-block" {>= "0.2"}
-  "mirage-block-unix" {>= "2.1.0"}
+  "mirage-block-unix" {>= "2.1.0" & < "2.5.0"}
```

Comment:
Does the constraint need to be this tight if the interface only changed in Mirage 3?